core/src/test/java/greencity/service/impl/SearchServiceImplTest.java

```diff
@@ -0,0 +1,40 @@
+package greencity.service.impl;
+
+import greencity.ModelUtils;
+import greencity.dto.search.SearchNewsDto;
+import greencity.service.EcoNewsService;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.mockito.InjectMocks;
+import org.mockito.Mock;
+import org.modelmapper.ModelMapper;
+import org.springframework.test.context.junit.jupiter.SpringExtension;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.mockito.ArgumentMatchers.anyString;
+import static org.mockito.Mockito.when;
+
+@ExtendWith(SpringExtension.class)
+class SearchServiceImplTest {
+
+    @InjectMocks
+    private SearchServiceImpl searchService;
+
+    @InjectMocks
+    private ModelMapper modelMapper;
+
+    @Mock
+    private EcoNewsService ecoNewsService;
+
+    @Test
+    void searchTest() {
```

Comment:
Bad test. Nothing is tested here. You new to mock repo not service

Reply:
Maybe I something not understands. 
`SearchServiceImpl.class` have only one field with `EcoNewsService`  and one method `search`, which only created a new object `SearchResponseDto`, and I in test mock this services ( `EcoNewsService` ) and check if the object is created correctly.

I added a new test for `EcoNewsServiceImpl` where I mock `EcoNewsRepo`, and checked if we correctly map object what get from DB;